packages/@blockprotocol/type-system-node/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@blockprotocol/type-system-node",
+  "version": "0.0.0",
```

Comment:
If we are ready to publish these packages, I'd version them as `0.0.1`. We can later use Changesets to prepare new versions.

Reply:
If we want to try merging packages before releasing, we can keep 0.0.0 for now.

Reply:
I don't think they're worth publishing in their current state at all as is. I think there'll be a separate PR like "Prep type-system packages for publishing" or something in the future. (I suspect when we've implemented the 4 ontology types and it's not just a convoluted, oversized URI validation library 😁) We can try and revisit merging them at that point, as well as making sure we're happy with the package.json's, descriptions, etc.